k6/panic.go

```diff
@@ -0,0 +1,37 @@
+package k6
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	k6common "go.k6.io/k6/js/common"
+)
+
+// Panic throws a k6 error, and before throwing the error, it finds the
+// browser process from the context and kills it if it still exists.
+// TODO: test.
```

Comment:
I think it will be useful to refactor the comment a bit, maybe something along the lines of:

```suggestion
// Panic will cause a panic with the given error which will shut
// the application down. Before panicking, it will find the
// browser process from the context and kill it if it still exists.
// TODO: test.
```